cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,19 @@
+import time
```

Comment:
Add one blank line between different import groups in the future to follow PEP-8